### PR TITLE
Removes unnecessary variables.

### PR DESCRIPTION
**Pull request by systemovich**
* Fixes typos.

### CODE REVIEW

File: src/Loader.php [hunk 1]
```diff
@@ -2,15 +2,15 @@
 
 namespace Dotenv;
 
-use Dotenv\Exception\InvalidFileException;
-use Dotenv\Exception\InvalidPathException;
+use Dotenv\Exception\InvalidFileException,
+    Dotenv\Exception\InvalidPathException;
```
Comment (GrahamCampbell) [inline src/Loader.php]: No.

**Comment by GrahamCampbell** [comment]
Lots of these changes are incorrect so I cannot accept this PR.

File: src/Loader.php (hunk 1 above)
Comment (systemovich) [inline src/Loader.php]: PSR-2?

**Comment by systemovich** [comment]
@GrahamCampbell, the only other problem I can see is the `@return` in the doc comment of some functions. What else did I miss?

**Pull request closed without merging**